GitUI/Editor/FileViewer.cs

```diff
@@ -608,7 +608,7 @@ private FileInfo GetFileInfo(string fileName)
             return new FileInfo(resolvedPath);
         }
 
-        public static string ToHexDump(byte[] bytes, StringBuilder str, int columnWidth = 8, int columnCount = 2)
+        public static string ToHexDump(byte[] bytes, StringBuilder str, int columnWidth = 8, int columnCount = 2, int maxBytes = 8 * 2 * 1024)
```

Comment:
How did you decide on 1024 lines?
Is there a reason to show, say, more than 50 or 100 lines?

Reply:
1024 bytes has no noticeable delay and is scroll able. 
As this is a hex dump it is reasonable with a binary number.
256 lines (0x1000 bytes) is another candidate.

Reply:
> I can't think how 1,000 of lines of hex would be of significant use to
anyone.
I suggest we do less

It could also be that GE detects a binary file incorrectly (a file ending used for other intentions than anticipated?), this would be a reserver 16KB files are just  some hundreds of lines of code.
This is not important so I changing this to 4KB, commenting better and removing the unused parameter.